.travis.yml

```diff
@@ -25,6 +25,7 @@ script:
 - yarn build
 - yarn test
 - yarn test:size
+- yarn shipjs:release
```

Comment:
It checks the following things to see if it can release it:

- see if the latest commit message starts with `chore: release v${currentVersion}`
- see if the current branch is `master` or `maint`.
  - If it's `master`, then it's coming from `develop`. It will release this, and merge it back to `develop`.
  - If it's `maint`, then it's coming from `maint`. It will release this.

If the conditions are met, then it releases.

Reply:
what happens when you merge inside master a branch which is not develop? is the release still happening? would it still assume it's coming from develop?

Reply:
The current mergeStrategy is

```js
  mergeStrategy: {
    toSameBranch: ["maint"],
    toReleaseBranch: {
      develop: "master"
    }
  },
```

It means things will flow like this: 
> develop -> releases/vx.y.z -> master -> back to develop

We will get a PR `releases/vx.y.z -> master`. When we merge it, it will be merged to `master`. And the CI service will pick up the commit, trigger a release and later merge it back to develop.

Does this answer to your question?